src/checkers/inference/VariableAnnotator.java

```diff
@@ -80,6 +78,7 @@
 import checkers.inference.util.ASTPathUtil;
 import checkers.inference.util.CopyUtil;
 import checkers.inference.util.InferenceUtil;
+import org.checkerframework.org.plumelib.util.IPair;
```

Comment:
Is it possible to use the normal, non-shaded plumelib artifact?
In many (all?) of the uses, the `IPair` here can be a separate `IPair` from the one that's used in the CF.
The disadvantage would be that, if we also shaded CFI, we would have two separate `IPair` classes.

Are any of the `IPair` uses actually from a CF API?

Maybe this is okay, but I think this is the first time we depend on a shaded artifact, so we should think about it, and document the reasoning somewhere.

Reply:
I have updated them all non-shaded plumelib artifact.